src/utils/formatting.ts

```diff
@@ -1,5 +1,6 @@
 import dayjs from "dayjs";
 import { ethers } from "ethers";
+import { BetHistory } from "../types/bets";
```

Comment:
unused import

Reply:
yeah, just spotted it once it built, removed just now :D